studio.ps1

```diff
@@ -4,6 +4,15 @@ $ErrorActionPreference = "Stop"
 . "$PSScriptRoot\common.ps1"
 . "$PSScriptRoot\extra_software.ps1"
 
+function Enable-NtpServer {
+    Write-Output "Enabling NTP Server on Studio VM..."
+    Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\W32Time\TimeProviders\NtpServer' -Name 'Enabled' -Value 1
+    Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\W32Time\Config' -Name 'AnnounceFlags' -Value 5
+    Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Services\W32Time\Parameters' -Name 'Type' -Value 'NTP'
+    Restart-Service w32time
+    New-NetFirewallRule -Name "NTP_Inbound" -DisplayName "Inbound NTP" -Protocol UDP -LocalPort 123 -Direction Inbound -Action Allow

```

Comment:
Why do we need `New-NetFirewallRule`, we disable the firewall at the first stage?